bam_qc_stats.cwl

```diff
@@ -0,0 +1,129 @@
+class: Workflow
+cwlVersion: v1.0
+id: bam_qc_stats
+label: bam_qc_stats
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
```

Comment:
This is missing some name spaces based on the tags used for Author and Organization.  Wee need to add to avoid seeing warning 
  dct: 'http://purl.org/dc/terms/'
  doap: 'http://usefulinc.com/ns/doap#'
  foaf: 'http://xmlns.com/foaf/0.1/'

Reply:
@murphycj2 probably in the new version of this we can add these tags